tests/test_feature_extraction.py

```
@@ -108,11 +112,19 @@
                                        imageDirPath = "tests/",
                                        roiNames = None,
-                                       outputDirPath = "tests/output/")
+                                       outputDirPath = "tests/NSCLC_Radiogenomics/results/")
     expected_path = "tests/output/features/radiomicfeatures_original_NSCLC_Radiogenomics.csv"
-    assert os.path.exists(expected_path)
+    assert os.path.exists(expected_path)
+
+
+def test_4DLung_radiomicFeatureExtraction_output(lung4DMetadataPath):
+    """Test output creation from radiomic feature extraction for RTSTRUCT dataset"""
+    actual = radiomicFeatureExtraction(lung4DMetadataPath,
```

Ruff (0.8.2)

Local variable `actual` is assigned to but never used

Remove assignment to unused variable `actual`

(F841)